src/telcameras_v2/migrations/0019_drop_old_views_20210209_1153.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+VIEWS = [
+    'cmsa_1h_count_view_v1',  # TODO: Check whether anybody still uses this view
+    'cmsa_15min',
+    'cmsa_15min_view_v2',
+    'cmsa_15min_view_v3',
+    'cmsa_15min_view_v4',
+    'cmsa_15min_view_v5',
+    'cmsa_15min_view_v6_realtime',
+    'cmsa_15min_view_v6',
+]
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('telcameras_v2', '0018_drop_old_tables_20200208_1555'),
+    ]
+
+    # Remove the old views which aren't used anymore
+    operations = []
+    for view in VIEWS:
+        if view != 'cmsa_15min_view_v6_realtime':
```

Comment:
waarom is deze rule nodig en halen we hem niet gewoon uit de lijst hier boven ?

Reply:
Omdat alleen die view geen materialized version heeft. Bij alle anderen moet ook een materialised versie van de view worden verwijderd.